server/channels/api4/team.go

```diff
@@ -816,7 +816,7 @@ func addTeamMembers(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	var appErr *model.AppError
 	var members []*model.TeamMember
-	if jsonErr := json.NewDecoder(r.Body).Decode(&members); jsonErr != nil {
+	if jsonErr := json.NewDecoder(r.Body).Decode(&members); jsonErr != nil || members == nil {
```

Comment:
Is this really needed? There's a check which verifies the length of a slice after this.